src/PoshArmDeployment/Public/ResourceProviders/Microsoft.Network/ApplicationGateway/Add-ArmApplicationGatewayHealthProbe.ps1

```diff
@@ -0,0 +1,55 @@
+function Add-ArmApplicationGatewayHealthProbe {
+    [CmdletBinding(SupportsShouldProcess = $true)]
+    [OutputType("ApplicationGateway")]
+    Param(
+        [PSTypeName("ApplicationGateway")]
+        [Parameter(Mandatory, ValueFromPipeline)]
+        $ApplicationGateway,
+        [ValidatePattern('^(\[.*\]|[a-zA-Z0-9-_.]*)$')]
+        [string]
+        $Name = "default",
+        [string]
+        [ValidateSet("Http", "Https")]
+        $Protocol = "Http",
+        [string]
+        $HostName,
+        [string]
+        $Path = "/",
+        [int]
+        $IntervalInSeconds = 30,
+        [int]
+        $TimeoutInSeconds = 30,
+        [int]
+        $UnhealthyThreshold = 3,
+        [switch]
+        $PickHostNameFromBackendHttpSettings,
```

Comment:
I feel like the default should be true, so we need to reverse that switch

Reply:
I followed Azure portal default values.
I'll change that